manifests/web_server.pp

```diff
@@ -10,6 +10,7 @@
   String  $ssl_chainbundle_cert = '',
   String  $ssl_cert_private_key = '',
   String  $irida_ip_addr = $ipaddress,
+  String  $irida_url_path = 'irida',
```

Comment:
I don't particularly like setting both this parameter, and the respective parameter in `init.pp` to a default of `irida`; I feel like we should just put it in one place. Does anyone have any ideas for setting a default for this parameter in only one location?

Reply:
Ultimately, it was decided that this pattern is acceptable.

An alternative that was considered was to have the `web_server` class include the `irida` class itself. This would have allowed `web_server` to access all of the parameters passed to `irida`, which would have been convenient but not followed separation of concerns as well. 

Another consideration was to only have the default set in one location, and have it be undefined in another. However it was decided that it is acceptable to have this default set in both locations, since each class is designed so that it can be used without the necessary inclusion of the other.